src/qibolab/backends.py

```diff
@@ -50,6 +52,45 @@ def __init__(self, platform):
         }
         self.compiler = Compiler.default()
 
+    @property
+    def qubits(self) -> list[Union[str, int]]:
+        """Returns the qubits in the platform."""
+        return list(self.platform.qubits)
+
+    @property
+    def connectivity(self) -> list[tuple[Union[str, int], Union[str, int]]]:
+        """Returns the list of connected qubits."""
+        return list(self.platform.pairs)
+
+    @property
+    def natives(self) -> list[str]:
+        """Returns the list of native gates supported by the platform."""
+        compiler = Compiler.default()
+        natives = [g.__name__ for g in list(compiler.rules)]
+
+        if "CZ" in natives and "CNOT" in natives:
+            for pair in self.connectivity:
+                cz = gates.CZ(*pair)
+                cnot = gates.CNOT(*pair)
+
+                if not self._is_gate_calibrated(cz, compiler):  # pragma: no cover
+                    natives.remove("CZ")
+                    break
+
+                if not self._is_gate_calibrated(cnot, compiler):
+                    natives.remove("CNOT")
+                    break
+
+        return natives
+
+    def _is_gate_calibrated(self, gate, compiler) -> bool:
+        """Helper method to check if a gate is calibrated."""
+        try:
+            compiler[gate.__class__](gate, self.platform)
```

Comment:
```suggestion
            compiler[type(gate)](gate, self.platform)
```
Avoid as much as possible to use dunders.